package.json

```diff
@@ -80,7 +81,6 @@
     "prettier": "^2.5.1",
     "react": "^17.0.2",
     "react-router-dom": "^6.0.0-beta.6",
-    "schema-dts": "^1.1.0",
```

Comment:
All peerDependencies needs to also be devDependencies so they are installed when developing the library.